core/src/main/java/io/confluent/rest/jetty/DoSFilter.java

```diff
@@ -1025,7 +1016,10 @@ public boolean addWhitelistAddress(@Name("address") String address) {
 
     private boolean addWhitelistAddress(List<String> list, String address) {
         address = address.trim();
-        return address.length() > 0 && list.add(address);
+        if (address.length() <= 0)
```

Comment:
Any reason to update this logic for the jetty change?

Reply:
I think this must have been pulled from Jetty 9 changes https://github.com/jetty/jetty.project/commit/de862c558b1b2cccf03d1f17f72a811ef8e18734#diff-8cf68b792fa7e539e2ad0c49fe7bcdb78953fff6cdbf26529c0d122dd6816bf3R1097-R1104

functionally, it's the exact same. `list.add` always returns `true`